packages/server/src/script/resendSignupEmails.ts

```diff
@@ -128,30 +156,39 @@ const logRegistrationPreviews = (registrations: WithId<RichStateInfo>[]) => {
 
 // Reads './data/conditions.ts' and resends emails to all voter sign ups that satisfy query.
 const main = async () => {
-    // Fetch registrations.
-    const registrations = await firestoreService.query<RichStateInfo>(conditions.query)
-    if (registrations == null || registrations.length == 0) {
-        console.log('No voters found for specified conditions.')
-        return
+    for (const query of conditions.queries) {
+        // Fetch registrations.
+        const registrations = await firestoreService.query<RichStateInfo>(query)
+        if (registrations == null || registrations.length == 0) {
+            console.log('No voters found for specified conditions.')
+            return
+        }
+
+        // Display registration previews.
+        logRegistrationPreviews(registrations)
+
+        // Get confirmation form user that emails should be sent.
+        const confirmation = `Would you like to resend emails to all of these voters and their election officials? ['Y' | 'N']\n`
+        const rl = readline.createInterface({
+            input: process.stdin,
+            output: process.stdout,
+        })
+        await new Promise(resolveInput => rl.question(confirmation, async (data) => {
+            if (data.toString() !== 'Y'){
+                console.log('You typed something other than \'Y\'. Exiting.')
+                rl.close()
+                rl.removeAllListeners()
+                resolveInput()
+            } else {
```

Comment:
Can we actually just do this for queries[0]?

Reply:
Done, thanks for the feedback!